frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices-mode/install-wizard.tsx

```diff
@@ -126,6 +126,10 @@ const CreateStorageClusterWizard: React.FC<CreateStorageClusterWizardProps> = ({
   const discoveryNodes = state.lvdIsSelectNodes ? state.lvdSelectNodes : state.lvdAllNodes;
 
   const { getStep, getIndex, getAnchor } = navUtils;
+  const hasConfiguredNetwork =
+    state.networkType === NetworkType.MULTUS
+      ? !!(state.publicNetwork || state.clusterNetwork)
+      : true;
```

Comment:
Why are we not doing any check for cluster network ?
Returning true always for other option might cause problem too.

Reply:
We have not checks for Cluster Network. And for https://github.com/openshift/console/pull/9541#pullrequestreview-711292259 we are doing this for attached-devices-mode.